packages/ida.plugin.scyllahide.vm/ida.plugin.scyllahide.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>ida.plugin.scyllahide.vm</id>
+    <version>1.4</version>
+    <description>ScyllaHide is an advanced open-source x64/x86 user mode Anti-Anti-Debug library.</description>
+    <authors>NtQuery</authors>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="idafree.vm" version="[8.3]" />
```

Comment:
Once you use the IDA user directory (instead of a concrete version), remove the version requirement here, as we don't need to require a specific version for our packages (only in cases where there is a minimum requirement because of a new feature for example, but this is not the case here).

Reply:
Since we're going to install the plugin to the global plugins directory I don't think we need the idafree package as a dependency.
Resolved at https://github.com/mandiant/VM-Packages/pull/701/commits/18dfd4eee0cb4d2b6cb5eaecbac49e43cfbe1dcf.